star-ros2/src/star_safety_monitor/include/star_safety_monitor/safety_monitor.hpp

```
@@ -0,0 +1,154 @@
+  STALE = 3
+};
+
+class SafetyMonitor : public rclcpp_lifecycle::LifecycleNode
+{
+public:
+  explicit SafetyMonitor(const rclcpp::NodeOptions & options = rclcpp::NodeOptions());
+  ~SafetyMonitor();
+
+  // Lifecycle callbacks
+  rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn
+  on_configure(const rclcpp_lifecycle::State & previous_state) override;
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)